packages/hashtower.sol/package.json

```diff
@@ -0,0 +1,50 @@
+{
+    "name": "hashtower.sol",
```

Comment:
Is it ok for you to use `@zk-kit/hashtower.sol` as a package name?

Reply:
I just added the missing `contracts/package.json` with `"name": "@zk-kit/hashtower.sol"` in it.

Reply:
Yes, it makes sense